Readme.md

```diff
@@ -77,6 +84,13 @@ docker run --rm -it -e SECRET_KEY=<YOUR_SECRET_KEY> -e DEBUG_ENV=1 -e CPPFLAGS="
 
 This should set you up for local development.
 
+### Installing Developer Packages
+Setup a virtualenv and run:
+
+```shell script
+CPPFLAGS="$(pg_config --cppflags)" LDFLAGS="$(pg_config --ldflags)" python3 -m pip install -r requirements_dev.txt 
```

Comment:
`CPPFLAGS="$(pg_config --cppflags)" LDFLAGS="$(pg_config --ldflags)"` this part is not needed for dev tools pkg

Reply:
I await your approval for the change.